api/Clipboard.json

```diff
@@ -74,7 +80,9 @@
           "support": {
             "chrome": [
               {
-                "version_added": "76"
+                "version_added": "76",
```

Comment:
Is this really a partial implementation? Reading the spec, it's not clear to me that implementations are obligated to support all MIME types. Maybe you have more familiarity with the spec. If it is not obligatory, then I'd guess we ought to redo this as a single entry, without partial and the notes are formatted as:

1. `"From version 76, the <code>image/png</code> MIME type is supported."`
2. `"Before version 76, images are not supported."`

If implementations are supposed to be supporting arbitrary MIME types (or some set of MIME types,) then I think the existing structure is fine, though I might revise the note for the current version to stand alone a little better, perhaps `"Only <code>image/png</code> MIME type images are supported."`

Reply:
For `write()`, this turns out to be browser specific. For `read()` it's not.

>These data types must be exposed by paste events if a corresponding native type exists on the clipboard. ([Link](https://w3c.github.io/clipboard-apis/#reading-from-clipboard))

I've changed the way I did `write()`, but not `read()`.